docs/source/release_notes.rst

```diff
@@ -9,6 +9,7 @@ Future Release
     * Fixes
     * Changes
         * Add autonormalize as an add-on library (:pr:`1840`)
+        * Add DateToHoliday Transform Primitive (:pr: `1846`)
```

Comment:
Seems like the release notes ci check is failing because the PR number is slightly wrong--should be 1848!

Reply:
ahh thanks! I was wondering what was wrong here